src/sdk/metrics/exporters/otlp.zig

```diff
@@ -18,7 +19,94 @@ const ManagedString = protobuf.ManagedString;
 const pbcommon = @import("../../../opentelemetry/proto/common/v1.pb.zig");
 const pbmetrics = @import("../../../opentelemetry/proto/metrics/v1.pb.zig");
 
-pub fn toProtobufMetric(
```

Comment:
This file used to be a helper for converting the internal metrics data model to the protobuf types.
It is now moved to an implementation of an exporter (see where we call it in `sdk/metrics/exporters.zig`).